scipy/stats/stats.py

```diff
@@ -3692,6 +3692,103 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+AlexanderGovernResult = namedtuple("AlexanderGovernResult", ("statistic",
+                                                             "pvalue"))
+
+
+def AlexanderGovern(*args):
```

Comment:
```suggestion
def alexandergovern(*args):
```
To fit with all the other [scipy.stats test](https://docs.scipy.org/doc/scipy/reference/stats.html#statistical-tests) names like `mannwhitneyu` and `brunnermunzel`.